test/cli/config.js

```diff
@@ -58,7 +58,7 @@ describe('config', () => runOnAndOff((thing) => {
     })
 
     it('set a config key with invalid json', () => {
-      return ipfs.fail('config foo {"bar:0} --json')
+      return ipfs.fail('config foo {"bar:0"} --json')
```

Comment:
I think this test is meant to fail!

Reply:
and still does!
if we don't close the quotes the stdin keeps open for manual input